.github/workflows/ci-actions-incremental.yml

```diff
@@ -165,7 +165,11 @@ jobs:
             GIB_IMPACTED=''
           fi
           echo "GIB_IMPACTED: ${GIB_IMPACTED}"
-          echo "::set-output name=impacted_modules::${GIB_IMPACTED//$'\n'/'%0A'}"
+          # three steps to retain linefeeds in output for other jobs
+          # (see https://github.com/github/docs/issues/21529 and https://github.com/orgs/community/discussions/26288#discussioncomment-3876281)
+          echo 'impacted_modules<<EOF' >> $GITHUB_OUTPUT
```

Comment:
You don’t need an equal sign in this case?

Reply:
I know it looks strange but it works, e.g. https://github.com/famod/ci-test/actions/runs/4128841110